api/v01/entities/vrp_input.rb

```diff
@@ -239,12 +239,16 @@ module VrpMisc
                                                                  shipment meetup
                                                                  minimum_duration_lapse maximum_duration_lapse
                                                                  force_first never_first force_end
+                                                                 vehicle_trips
                                                                  vehicle_group_duration vehicle_group_duration_on_weeks
                                                                  vehicle_group_duration_on_months vehicle_group_number],
                     desc: 'Relations allow to define constraints explicitly between activities and/or vehicles.
                            It could be the following types: same_route, sequence, order, minimum_day_lapse, maximum_day_lapse,
-                           shipment, meetup, minimum_duration_lapse, maximum_duration_lapse')
-    optional(:lapse, type: Integer, desc: 'Only used for relations implying a duration constraint : minimum/maximum day lapse, vehicle group durations...')
+                           shipment, meetup, minimum_duration_lapse, maximum_duration_lapse, vehicle_trips')
+    optional(:lapse, type: Integer, desc:
+      'Only used for relations implying a duration constraint.
+      Lapse expressed in days for minimum/maximum day lapse,
+      in seconds for vehicle_group_durations or vehicle_trips.')
```

Comment:
Do we want this style? How does it look in the auto generated documentation? 

I think it is better if we leave the description parsing to grape-swagger and disable line width cop for this file because thanks to all descriptions being a single line, all parameter definitions come one after another and it is very easy to follow/skim through the parameters; otherwise, it will look scattered I think. If this is what we want we should do it for every definition; if not, it is like reading a book with different font for every word -- i.e., hard on the eyes :eyes: 

What do you think @fonsecadeline, @braktar?

Reply:
@fonsecadeline raised a valid issue "the `coerce_with` logic comes after the `description` which hides them off screen", I think we can make sure that the description is always last. What do you guys think, other suggestions?

Reply:
I apparently started my review before you sent you comment Adeline :D

Reply:
I don't think it is that hard to follow `description comes always last`..

Maybe we should compare it visually? I tested and my vote is on keeping the description always at the last position and dis-activating the line with cop on this file.

Reply:
so it turns out the generated docs are not correct :facepalm: so we decided to get the best of two worlds... 

description is always at last, always on a separate line of its own but even if it violates the line with cop we don't cut it.